Sources/AccioKit/Commands/Protocols/DependencyInstaller.swift

```diff
@@ -83,5 +89,61 @@ extension DependencyInstaller {
 
         try XcodeProjectIntegrationService.shared.handleRemovedTargets(keepingTargets: appTargets)
         try bash("rm -rf '\(Constants.temporaryFrameworksUrl.path)'")
+
+        try ResolvedManifestCachingService(sharedCachePath: sharedCachePath).cacheResolvedManifest(
+            at: URL(fileURLWithPath: workingDirectory).appendingPathComponent("Package.resolved"),
+            with: parsingResults.flatMap {
+                $0.frameworkProducts.map {
+                    CachedFrameworkProduct(swiftVersion: Constants.swiftVersion, libraryName: $0.libraryName, commitHash: $0.commitHash, platform: $0.platformName)
+                }
+            }
+        )
+    }
+
+    func attemptUncachingAllRequiredFrameworks(
```

Comment:
Consider giving this function a name indicating an action like `tryLoadingAllRequiredFrameworksFromCache`

Reply:
How is this different from the current naming? What additional information does it hold or what misleading wording does it fix? Could you please further explain this?

Reply:
It seams simpler to understand for me also the action of `loading` is decoupled from the `cache`. But thats a personal thing you can stick with your naming :)

Reply:
Okay, I renamed to `attemptLoadingRequiredFrameworksFromCache`, keeping the `attempt` keyword instead of `try` since using try would then read `try tryLoading...` which is weird (since the function is throwing).

Reply:
Fixed.